pkg/authorization/bootstrap/policy.go

```diff
@@ -76,6 +80,12 @@ func clusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("delete", "update", "get").Groups(tenancy.GroupName).Resources("workspaces").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: SystemExternalLogicalClusterAdmin},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule("delete", "update", "get").Groups(core.GroupName).Resources("logicalclusters", "logicalclusters/status").RuleOrDie(),
+			},
```

Comment:
@sttts and/or @s-urbaniak I added a new group as discussed but this rule doesn't seem to be enough to get logical clusters at cluster scope:

```
I0316 17:47:58.446040  572728 httplog.go:131] "HTTP" verb="GET" URI="/clusters/1jys1xdusxcx96v2/apis/core.kcp.io/v1alpha1/logicalclusters/cluster" latency="538.409µs" userAgent="kcp/v1.24.3+kcp (linux/amd64) kubernetes/b5751ce/kcp-workspace+root" audit-ID="a0d98940-2691-4798-8f98-a5148703ba03" srcIP="10.19.133.66:38396" resp=403
E0316 17:47:58.446644  572728 workspace_controller.go:225] "kcp-workspace" controller failed to sync "root|e2e-workspace-hktgw", err: logicalclusters.core.kcp.io "cluster" is forbidden: User "external-logical-cluster-admin" cannot get resource "logicalclusters" in API group "core.kcp.io" at the cluster scope: access denied
```

Any pointers on how to make the authz work without the short-circuit logic in the Authorizer like the regular `logical-cluster-admin` ?

Reply:
After debugging with @s-urbaniak we concluded that the new `external-logical-cluster-admin` group still needs to be special-cased in the admission and content authorizer code (like the existing `logical-cluster-admin`), however we can avoid the special-casing in the requiredgroups authorizer, and the bootstrap policy rules are significantly more restrictive in the external case.